src/tests/resetPassword.test.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { expect, describe, it } from 'vitest';
+import { render } from '../utilis/testUtilis';
+import ResetPassword from '../pages/ResetPassword';
+
+describe('Reset Password component', () => {
+	it('should dispay password input field', () => {
+		render(<ResetPassword />);
+		const PasswordField = screen.getByPlaceholderText('password');
+		expect(PasswordField).toBeInTheDocument();
+	});
+	it('should dispay  confirm password input field', () => {
+		render(<ResetPassword />);
+		const confirmPasswordField = screen.getByPlaceholderText('password');
+		expect(confirmPasswordField).toBeInTheDocument();
+	});
+
+	it('should dispay reset password button ', () => {
+		render(<ResetPassword />);
+		const button = screen.getByRole('button');
+		expect(button).toBeInTheDocument();
+	});
+
+	it('should render the ResetPassword component', () => {
+		render(<ResetPassword />);
+		const component = screen.getByRole('heading');
+		expect(component).toBeInTheDocument();
```

Comment:
- Add more test scenarios other than component `render` and element `toBeInTheDocument`.